core/domain/stats_services.py

```diff
@@ -421,15 +476,22 @@ def update_exp_issues_for_new_exp_version(
 
     playthrough_ids = list(itertools.chain.from_iterable(
         issue.playthrough_ids for issue in exp_issues.unresolved_issues))
-    playthrough_models = (
+    playthrough_models_with_none = (
```

Comment:
Should we change the name since this list will not contain 'None'?

Reply:
The methods `update_timestamps_multi` and `put_multi` can only accept `non-none lists`. So, if we keep the original name then at line 543 and 544 it will contradict the argument type because then `playthrough_models` have List[Optional[]] type.

Reply:
So what I meant to say was, that we are asserting that `playthrough_model is not None`.
So should we consider renaming `playthrough_models_with_none` to  something meaningful since it wont contain `None`?

Ditto elsewhere

Reply:
So what I meant to say was, that we are asserting that playthrough_model is not None.
So should we consider renaming playthrough_models_with_none to something meaningful since it wont contain None?

Ditto elsewhere

Reply:
Done, we can change the name here --  because we are updating the model and then putting it inside the method. 
So, older ones are playthrough_models and new ones are updated_playthrough_models.